docs/jargon-mesh.md

```diff
@@ -0,0 +1,658 @@
+---
+id: terminology
+title: Temporal's core terminology
+sidebar_label: Terminology
+---
+
+import Mermaid from '@theme/Mermaid';
+
+## Workflow
+
+A fault-oblivious stateful function that orchestrates Activities.
+
+- A Workflow has full control over which [**Activities**](#activity) are executed, and in which order.
+- A Workflow must not affect the external world directly, only through [**Activities**](#activity).
+- What makes Workflow code a Workflow is that its state is preserved by Temporal.
```

Comment:
Are we with this saying that it i impossible to write "stateless" workflows with Temporal?